libr/core/cmd_anal.c

```diff
@@ -5032,22 +5029,6 @@ static void cmd_anal_trace(RCore *core, const char *input) {
 		// XXX: not yet tested..and rsc dwarf-traces comes from r1
 		r_core_cmd (core, "at*|rsc dwarf-traces $FILE", 0);
 		break;
-	case '+': // "at+"
```

Comment:
The reason for having at and dt is because at works with esil without debugging, just emulating. Are you testing that this is still working while doing the changes? Can you add tests?